gempython/tools/optohybrid_user_functions_uhal.py

```diff
@@ -171,7 +171,9 @@ def broadcastRead(device,gtx,register,mask=0xff000000,debug=False):
         time.sleep(0.1)
         pass
 
-    return readBlock(device,"%s.Results"%(baseNode),24)
+    # bitcount = bits not set in mask
+    bitcount = 24
```

Comment:
why it is hard-coded here? If it is a temp plug before we implement proper masking, shouldn't it be placed where we are going to get the mask?

Reply:
As you say, it is a placeholder to have me remember to actually implement it (if we decide to do so in #146)